internal/app/app.go

```diff
@@ -221,7 +221,7 @@ func (p *PolicyAutomationApp) evaluateClusters(regoPackageBases []string) error
 	}
 	// create a PolicyAgent client instance
 	pa := policy.NewPolicyAgent(p.ctx)
-	p.out.ColorPrintf("[light_gray][bold]Parsing REGO policies...\n")
+	p.out.ColorPrintf("\u2139 [light_gray][bold]Parsing REGO policies...\n")
```

Comment:
maybe we can define some named const values for icons?

Reply:
Good idea, I've added `icons.go` with consts starting with `ICON_xx`.
However in future we should think about automatic replacement method as we have now with colors :-)